lib/DDG/Meta/Data.pm

```diff
@@ -122,6 +125,26 @@ sub get_ia {
     return $m;
 }
 
+sub _satisfy {
+	my ($ia, $by, $lookup) = @_;
+	return $by->($ia, $lookup) if ref $by eq 'CODE';
+	my $pby = $ia->{$by};
+	ref $lookup eq 'CODE' ? $lookup->($pby) : $pby eq $lookup;
+}
+
+# get_ias(repo => 'goodies', dev_milestone => 'live'...)
+# Lookups combine as an AND operation.
+sub get_ias {
+	my (@lookups) = @_;
+	my @results = @ia_container;
+	foreach (pairs @lookups) {
```

Comment:
@zachthompson Btw, it feels weird to have to pass in an ARRAY - what's wrong with having a list?

`filter_ias(id => 'calculator')` vs `filter_ias([id => 'calculator'])`; why would we favor the latter?